src/ior.c

```diff
@@ -1794,53 +1811,39 @@ static IOR_offset_t *GetOffsetArrayRandom(IOR_param_t * test, int pretendRank,
         }
         srand(seed);
 
-        fileSize = test->blockSize * test->segmentCount;
-        if (test->filePerProc == FALSE) {
-                fileSize *= test->numTasks;
-        }
+        /* count needed offsets */
+        offsets = (test->blockSize / test->transferSize) * test->segmentCount;
 
-        /* count needed offsets (pass 1) */
-        for (i = 0; i < fileSize; i += test->transferSize) {
-                if (test->filePerProc == FALSE) {
-                        // this counts which process get how many transferes in
-                        // a shared file
-                        if ((rand() % test->numTasks) == pretendRank) {
-                                offsets++;
-                        }
-                } else {
-                        offsets++;
-                }
-        }
+        /* start with same array of offsets as in sequential case */
+        offsetArray = GetOffsetArraySequential(test, pretendRank);
```

Comment:
Here we are in the random branch?  But we start with sequential?  Does this make it less random than it used to be?